sdk/communication/communication-sms/src/utils/uuid.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { generateUuid } from "@azure/core-http";
+
+// This is used as a workaround to be able to stub generateUuid
+// during testing.
+export class Uuid {
```

Comment:
Classes with static functions are discouraged in TS/JS, export a const function instead.

Reply:
this is specifically for sinon stubbing. sinon can't stub an exported const function. i pulled this workaround from the tables sdk

Reply:
Got it now, this makes sense.